tests/testthat/test-get_code.R

```diff
@@ -195,63 +238,17 @@ testthat::test_that("@linksto tag indicate affected object if object is assigned
   )
 })
 
-testthat::test_that(
-  "get_code with datanames can extract the code when function creates an object which is used only on rhs",
-  {
-    code <- c(
-      "data(iris)",
-      "iris2 <- head(iris)"
-    )
-    tdata <- eval_code(teal_data(), code)
-    datanames(tdata) <- c("iris2")
-    testthat::expect_identical(
-      get_code(tdata, datanames = "iris2"),
-      paste("data(iris)", "iris2 <- head(iris)", sep = "\n")
-    )
-  }
-)
-
-testthat::test_that("get_code with datanames can extract the code when using <<-", {
-  code <- c(
-    "a <- 1",
-    "b <- a",
-    "b <<- b + 2"
-  )
+testthat::test_that("get_code extracts a line with assign where object is passed as character", {
```

Comment:
Please refine this description. "object is passed as character" is not readily understandable.

Reply:
this was a redundant test that was already covered in other test so I decided to remove f9e05c015dc35525702e17191feaa1ee4773b280